packages/tsx-editor/src/transpiler/exampleParser.ts

```diff
@@ -93,7 +93,7 @@ export function tryParseExample(example: string, supportedPackages: IBasicPackag
 
 /** @internal */
 export function _tryParseExample(example: string, supportedPackages: string[]): IExampleInfo | string {
-  const possibleComponents = example.match(new RegExp(COMPONENT_NAME_REGEX, 'gm'));
+  const possibleComponents = example.match(new RegExp(COMPONENT_NAME_REGEX.source, 'gm'));
```

Comment:
Thanks for noticing this! Sorry about the extra trouble but would you mind putting this in a separate PR (or I can)? I'm not familiar enough with Autofill to be comfortable approving that part of the change, but this part is fine and should really go in ASAP.

Reply:
Sure! just created [PR 10611](https://github.com/OfficeDev/office-ui-fabric-react/pull/10611), I'll get rid of these changes here